RecoEcal/EgammaClusterProducers/python/particleFlowSuperClusterECALBox_cfi.py

```diff
@@ -0,0 +1,76 @@
+import FWCore.ParameterSet.Config as cms
+
+particleFlowSuperClusterECALBox = cms.EDProducer("PFECALSuperClusterProducer",
+    # verbosity
+    verbose = cms.untracked.bool(False),
+    # clustering type: "Box" or "Mustache"
+    ClusteringType = cms.string("Box"),
+    # energy weighting: "Raw", "CalibratedNoPS", "CalibratedTotal"
+    EnergyWeight = cms.string("Raw"),
+
+    # this overrides both dphi cuts below if true!
+    useDynamicDPhiWindow = cms.bool(False),
+
+    # PFClusters collection
+    PFClusters = cms.InputTag("particleFlowClusterECAL"),
+    ESAssociation = cms.InputTag("particleFlowClusterECAL"),
+    BeamSpot = cms.InputTag("offlineBeamSpot"),
+
+    PFBasicClusterCollectionBarrel = cms.string("particleFlowBasicClusterECALBarrel"),
+    PFSuperClusterCollectionBarrel = cms.string("particleFlowSuperClusterECALBarrel"),
+    PFBasicClusterCollectionEndcap = cms.string("particleFlowBasicClusterECALEndcap"),
+    PFSuperClusterCollectionEndcap = cms.string("particleFlowSuperClusterECALEndcap"),
+    PFBasicClusterCollectionPreshower = cms.string("particleFlowBasicClusterECALPreshower"),
+    PFSuperClusterCollectionEndcapWithPreshower = cms.string("particleFlowSuperClusterECALEndcapWithPreshower"),
+
+    # use preshower ?
+    use_preshower = cms.bool(True),
+
+    # are the seed thresholds Et or Energy?
+    seedThresholdIsET = cms.bool(True),
+
+    # regression setup
+    useRegression = cms.bool(False), # regressions are mustache only
+    regressionConfig = cms.PSet(
+        regressionKeyEB = cms.string('pfscecal_EBCorrection_offline_v2'),
+        uncertaintyKeyEB = cms.string('pfscecal_EBUncertainty_offline_v2'),
+        regressionKeyEE = cms.string('pfscecal_EECorrection_offline_v2'),
+        uncertaintyKeyEE = cms.string('pfscecal_EEUncertainty_offline_v2'),
+        vertexCollection = cms.InputTag("offlinePrimaryVertices"),
+        ecalRecHitsEB = cms.InputTag('ecalRecHit','EcalRecHitsEB'),
+        ecalRecHitsEE = cms.InputTag('ecalRecHit','EcalRecHitsEE'),
+        applySigmaIetaIphiBug = cms.bool(False)
+    ),
+
+    # threshold for final SuperCluster Et
+    thresh_SCEt = cms.double(4.0),
+
+    # threshold in ECAL
+    thresh_PFClusterSeedBarrel = cms.double(3.0),
+    thresh_PFClusterBarrel = cms.double(0.5),
+
+    thresh_PFClusterSeedEndcap = cms.double(5.0),
+    thresh_PFClusterEndcap = cms.double(0.5),
+
+    # window width in ECAL
+    phiwidth_SuperClusterBarrel = cms.double(0.28),
+    etawidth_SuperClusterBarrel = cms.double(0.04),
+
+    phiwidth_SuperClusterEndcap = cms.double(0.28),
+    etawidth_SuperClusterEndcap = cms.double(0.04),
+
+    # threshold in preshower
+    thresh_PFClusterES = cms.double(0.),
```

Comment:
@slava77, do you mean that it was present only implicitly in `particleFlowSuperClusterECALBox` (via `fillDescriptions()`) but not explicitly ?

Reply:
Since `particleFlowSuperClusterECALBox` was almost fully defined anyway, I prefer to list here all parameters and their values.

I agree it can be simplified importing particleFlowSuperClusterECALMustache_cfi and setting only the different parameters, but I would leave it to a subsequent PR.